src/bin/ledger-tool.rs

```diff
@@ -17,17 +21,38 @@ fn main() {
                 .required(true)
                 .help("use DIR for ledger location"),
         )
+        .subcommand(SubCommand::with_name("print").about("Print the ledger"))
+        .subcommand(SubCommand::with_name("verify").about("Verify the ledger"))
         .get_matches();
 
     let ledger_path = matches.value_of("ledger").unwrap();
+    let entries = match read_ledger(ledger_path) {
```

Comment:
As I recall, the default error message from the `?` notation is pretty decent here. I've switched to `main() -> Result` on other CLIs: https://github.com/solana-labs/solana/blob/master/src/bin/keygen.rs#L14

Reply:
I think it's pretty gnarly myself, notably it doesn't tell you *what* file or directory was not found so you need to think harder unnecessarily.

Contrast:
```bash
$ cargo run --bin solana-ledger-tool -- --ledger garbage_path
    Finished dev [unoptimized + debuginfo] target(s) in 0.24s
     Running `target/debug/solana-ledger-tool --ledger garbage_path`
Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```
vs.
```bash
$ cargo run --bin solana-ledger-tool -- --ledger garbage_path
    Finished dev [unoptimized + debuginfo] target(s) in 0.26s
     Running `target/debug/solana-ledger-tool --ledger garbage_path`
Failed to open ledger at garbage_path: No such file or directory (os error 2)
```

Reply:
Yeah, that's really bad. Not sure how I saw something different.